bigquery/requirements.txt

```diff
@@ -1,8 +1,9 @@
 cachetools==4.1.0
 certifi==2020.4.5.2
 chardet==3.0.4
-elasticsearch==6.1.1
-elasticsearch-dsl==6.2.1
+elasticsearch==7.7.0
```

Comment:
At a quick glance, it looks to me like the library versions mirror the ES versions.
See https://pypi.org/project/elasticsearch/#history.
Our ES deployment is 7.7.1